### PR TITLE
[DOPS-2648] migrate sora-card-android to jenkins libs

### PR DESCRIPTION
**Pull request by baotn166**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,69 +1,36 @@
...
+  [$class: 'StringBinding', credentialsId: 'SORA_CARD_KYC_USERNAME_TEST', variable: 'SORA_CARD_KYC_USERNAME_TEST'],
+  [$class: 'StringBinding', credentialsId: 'SORA_CARD_KYC_PASSWORD_PROD', variable: 'SORA_CARD_KYC_PASSWORD_PROD'],
+  [$class: 'StringBinding', credentialsId: 'SORA_CARD_KYC_PASSWORD_TEST', variable: 'SORA_CARD_KYC_PASSWORD_TEST'],
+  [$class: 'StringBinding', credentialsId: 'SORA_BACKEND_DEBUG', variable: 'SORA_BACKEND_DEBUG'],
+  [$class: 'StringBinding', credentialsId: 'SORA_BACKEND_RELEASE', variable: 'SORA_BACKEND_RELEASE']
+]
+
+new org.soramitsu.mainLibrary().call(
+  agentLabel: "android",
+  skipSonar: true,
```
Comment (wpi86) [inline Jenkinsfile:25]: Why?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,69 +1,36 @@
...
+  skipDojo: true,
```
Comment (wpi86) [inline Jenkinsfile:26]: Why?

**Review by wpi86 — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (baotn166) [reply Jenkinsfile:25]: we have another ticket to enable, this is just for migration

File: Jenkinsfile (hunk 2 above)
Comment (baotn166) [reply Jenkinsfile:26]: we have another ticket to enable, this is just for migration

**Review by wpi86 — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]

**Review by arvifox — APPROVED** [review]

**Pull request merged**